test/helpers/index.js

```diff
@@ -11,10 +11,11 @@ export function getTagPositions(renderedComponent) {
   const outputTags = TestUtils.scryRenderedDOMComponentsWithTag(
     renderedComponent, 'li'
   );
-  const [tagC, tagB, tagA] = outputTags;
-  return {
-    a: tagA.getBoundingClientRect(),
-    b: tagB.getBoundingClientRect(),
-    c: tagC.getBoundingClientRect(),
-  };
+
+  // returns { a: ClientRect, b: ClientRect, c: ClientRect }
+  return ['a', 'b', 'c'].reduce((acc, key) => {
+    // eslint-disable-next-line no-param-reassign
+    acc[key] = outputTags.find(el => el.id === key).getBoundingClientRect();
```

Comment:
I'd do this the functional no-mutation way:

```js
return ['a', 'b', 'c'].reduce((acc, key) => ({
  ...acc,
  [key]: outputTags.find(el => el.id === key).getBoundingClientRect(),
}), {});
```

I forget if we've enabled the object rest/spread babel transform... if not, then the existing solution is fine (also cool with me if you want to add it)